src/stores/data.js

```diff
@@ -63,7 +67,11 @@ function getRow(dataInfo, key) {
     symbol: dataInfo.symbol,
     owner: dataInfo.owner_addr,
     lastRoundAPY: dataInfo.lastRoundAPY,
-    roundapy: dataInfo.apy,
+    lastRoundAPR: dataInfo.lastRoundAPR,
+    lastRoundYield: dataInfo.lastRoundAPR / 52,
+    roundAPY: dataInfo.apy,
+    roundAPR: dataInfo.apr,
+    roundYield: dataInfo.apr / 52,
```

Comment:
I don't think this is right. This APR is already the earning for the current round, I don't think we should divide it by 52 since it doesn't do the calculation for 1 year.